test/Makefile

```diff
@@ -122,6 +122,7 @@ $(INITRAMFS)/benchmark/bin:
 	@cp /usr/local/leveldb/benchmark/db_bench $@
 	@mv $@/db_bench $@/db_bench_leveldb
 	@cp /usr/local/leveldb/benchmark/db_bench_sqlite3 $@
+	@cp /usr/local/benchmark/schbench $@
```

Comment:
Adding this line will fail most CI tests. I suggest adding it later when the new Docker image is built.

Reply:
> Does it mean I'd have to update the Dockerfile and bump the version in one PR 

Yes, that would be great.